designsystem.digdir.no/components/Header/Header.tsx

```diff
@@ -43,36 +65,22 @@ const Header = () => {
             )}
           </button>
           <ul className={cn(classes.menu, { [classes.active]: open })}>
-            <li className={classes.item}>
-              <Link
-                href='/god-praksis'
-                className={
-                  router.pathname == '/god-praksis' ? classes.active : ''
-                }
-              >
-                God praksis
-              </Link>
-            </li>
-            <li className={classes.item}>
-              <Link
-                href='/grunnleggende'
-                className={
-                  router.pathname == '/grunnleggende' ? classes.active : ''
-                }
-              >
-                Grunnleggende
-              </Link>
-            </li>
-            <li className={classes.item}>
-              <Link
-                href='/komponenter'
-                className={
-                  router.pathname == '/komponenter' ? classes.active : ''
-                }
+            {menu.map((item, index) => (
+              <li
+                className={classes.item}
+                key={index}
```

Comment:
Using the index can cause flaky behaviour, so could we use the URL as a key instead?
```suggestion
                key={item.url}
```

Since the menu is a hard-coded array and there is no possibility to delete an item within the menu, the index should be fine to use. But I think we should try to avoid it anyway. :)